src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -112,25 +112,13 @@ void groupLoop(const std::vector<const Opm::Group*>& groups,
     }
 }
 
-template < typename T>
-std::pair<bool, int > findInVector(const std::vector<T>  & vecOfElements, const T  & element)
+template <typename T>
+std::optional<int> findInVector(const std::vector<T>  & vecOfElements, const T  & element)
```

Comment:
Unless there is a very good reason to use `int` I would suggest using `std::size_t` for an index.